configs/forseti_conf.yaml.in

```diff
@@ -55,6 +55,9 @@ inventory:
     # Valid values are: debug, info, warning, error
     loglevel: info
 
+    # Number of days to retain inventory data, 0 to keep forever
+    retention_days: 30
+
```

Comment:
Can you also please update the [forseti_conf.yaml.sample](https://github.com/GoogleCloudPlatform/forseti-security/blob/dev/configs/forseti_conf.yaml.sample) file?